tests/unit/amo/components/TestAddonCompatibilityError.js

```diff
@@ -40,50 +44,86 @@ describe(__filename, () => {
     });
   };
 
-  function render({ ...props }) {
+  const getClientCompatibilityNonFirefox = () => {
+    return createFakeClientCompatibility({
+      compatible: false,
+      downloadUrl: DOWNLOAD_FIREFOX_BASE_URL,
+      reason: INCOMPATIBLE_NOT_FIREFOX,
+    });
+  };
+
+  const getClientCompatibilityIncompatible = (compatibilityProps = {}) => {
```

Comment:
Maybe you could call this function `makeGetClientCompatibilityIncompatible` because it returns a function (which is different than the other `getClientXX` functions).